web/src/engine/osk/src/input/mouseDragOperation.ts

```diff
@@ -1,7 +1,56 @@
-import InputEventCoordinate from './inputEventCoordinate.js';
-
 type MouseHandler = (this: GlobalEventHandlers, ev: MouseEvent) => any;
 
+/**
+ * Represents the current location of the current cursor / touchpoint during
+ * an ongoing contact-point event series.  This class standardizes to .pageX
+ * (document) coordinates, rather than .clientX (viewport) coordinates.
+ */
+class InputEventCoordinate {
```

Comment:
This comes straight out of the previous file in GitHub's changed-file listings.  I've simplified it to include only the functions that are still utilized.

Reply:
The remaining bits are 'small enough' that it's probably not worth the effort at the immediate moment to shift paradigms for this file's exposed class.